package.json

```diff
@@ -6,72 +6,70 @@
     "easing",
     "ember-addon"
   ],
+  "repository": "https://github.com/peopleconnectus/ember-concurrency-scroll.git",
   "license": "MIT",
   "author": "Jake Bixby",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
-  "repository": "https://github.com/peopleconnectus/ember-concurrency-scroll.git",
   "scripts": {
     "build": "ember build",
-    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "lint:hbs": "ember-template-lint .",
```

Comment:
Can you either target the template files in this app/addon folders directly on this line or add `*` to the end of `/`s like @jakebixbyavalara recommended. Cause as it stands now the `lint:hbs` command is failing due to it incorrectly testing the node_module template files:

![image](https://user-images.githubusercontent.com/1480201/60922119-e2f06700-a250-11e9-97a2-1ab0a58a42d0.png)

Reply:
The line is as proposed by ember-cli and it works like this in other projects.

See: https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/package.json